plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -380,8 +385,8 @@ func TestAdmitPreferNonmutating(t *testing.T) {
 		},
 		"pod should prefer non-mutating PSP on update": {
 			operation:             kadmission.Update,
-			pod:                   unprivilegedRunAsAnyPod.DeepCopy(),
-			oldPod:                changedPod.DeepCopy(),
+			pod:                   podWithSC.DeepCopy(),
+			oldPod:                changedPodWithSC.DeepCopy(),
```

Comment:
nit: I think these names are backwards... `changedPod*` sohuld be the new one, not the "oldPod". Same for other test cases.

Reply:
Tried to follow the existing pattern there, but it confused me as well.

Reply:
fixed